### PR TITLE
Testing To Do list - part 2

### PR DESCRIPTION
**Pull request by anasbassi**
In this project, the following requirements were implemented.
- Linters were installed correctly,
- the edit and clear all functions were tested,
- the HTML DOM manipulation and update storage were mocked and tested,
- Describe method was used to group the tests.
- a descriptive readme was added.

### CODE REVIEW

File: src/__test__/edit.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  editText.type = 'text';
+  editText.className = 'editText';
+  editText.value = task.textContent;
+  taskContainer.replaceChild(editText, task);
+  editText.value = 'new task';
+  taskContainer.replaceChild(task, editText);
+  task.textContent = editText.value;
+};
+
+test('test edit function', () => {
```
Comment (vechicin) [inline src/__test__/edit.test.js:12]: Please make sure you group this test, even if it is an individual one, within a `describe()` method as it is a project requirement.

Screen Shot 2022-07-14 at 8 45 42

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Lizdev-05 and @anasbassi ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] All tests are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @anasbassi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Lizdev-05** [comment]
Thanks for reviewing and approving our project, we appreciate it.

**Pull request merged**